Classes/Issues/Issue+IssueType.swift

```diff
@@ -93,32 +93,56 @@ extension IssueOrPullRequestQuery.Data.Repository.IssueOrPullRequest.AsIssue: Is
                 }
             } else if let unlabeled = node.asUnlabeledEvent,
                 let date = unlabeled.createdAt.githubDate {
-                let model = IssueLabeledModel(
-                    id: unlabeled.fragments.nodeFields.id,
-                    actor: unlabeled.actor?.login ?? Constants.Strings.unknown,
-                    title: unlabeled.label.name,
-                    color: unlabeled.label.color,
-                    date: date,
-                    type: .removed,
-                    repoOwner: owner,
-                    repoName: repo,
-                    width: width
-                )
-                results.append(model)
+                    let newDetails = IssueLabeledModel.LabelDetails(
+                        id: unlabeled.fragments.nodeFields.id,
+                        title: unlabeled.label.name,
+                        color: unlabeled.label.color,
+                        date: date,
+                        type: .removed)
+                    var removed = [IssueLabeledModel.LabelDetails]()
+                    var added = [IssueLabeledModel.LabelDetails]()
+
+                    if let lastModel = results.last as? IssueLabeledModel,
+                        lastModel.actor == unlabeled.actor?.login {
+                        added = lastModel.added
+                        removed = lastModel.removed
+                        removed.append(newDetails)
```

Comment:
`newDetails` only gets appended in this block, which is why it breaks for events with a single label